Common/Kernel/CommonValidations.cs

```diff
@@ -13,5 +13,16 @@ public static void ValidateId(int id)
                 throw new Exception(Errors.IdCanNotBeLessThanZero);
             }
         }
+        
+        public static void ValidateEmail(string email)
+        {
+
+            var addr = new MailAddress(email);             

```

Comment:
Without try catch this will throw unhandled exception and whole system will be down

Reply:
fixed